ProjectSettings/TagManager.asset

```diff
@@ -17,7 +17,7 @@ TagManager:
   - Ink
   - Enemy
   - Player Party
-  - 
+  - post_processing
```

Comment:
Why not in the existing and standard TransparentFX layer?

Reply:
updated